### ISSUE TITLE
ANMN T and S should not harvest T from ADCPs

### ISSUE THREAD

**Issue by ggalibert** [aodn/content#271]
As a request from Dave Hugh and others expressed during the last IMOS plenary meeting:
> The temperature returned here is used primarily for internal ADCP operations and is not calibrated. It should not form part of the time series.

Impacts harvester ANMN_TS_TIMESERIES.

**Comment by ggalibert** [comment]
This PR: https://github.com/aodn/chef/pull/469 fixes part of the issue. Historical files already harvested need to be manually de-indexed from database.

**Comment by mhidas** [comment]
We should do the same for the ABOS TS collection: https://github.com/aodn/chef/pull/478

@ggalibert @smancini

**Comment by ggalibert** [comment]
Can we close this @mhidas ?

**Comment by ggalibert** [comment]
I believe historical files have been manually de-indexed right?

**Comment by mhidas** [comment]
Yes, I had done that, but now there are ADCP files in there again. As it turns out, we forgot to update the talend config in chef-private, which controls what happens in RC & prod (The chef PRs referenced above only apply to the PO box).

This is fixed by https://github.com/aodn/chef-private/pull/2506 and https://github.com/aodn/chef-private/pull/2507. Once those are merged and deployed to prod, I will repeat the un-haravesting of any files that have made it in there.

**Comment by mhidas** [comment]
I have repeated the un-harvest of ADCP files from ANMN_TS. 
ABOS_TS has a problem (https://github.com/aodn/harvesters/issues/567) which will need to be resolved before we can do the same there.
